### PR TITLE
Move react-scripts dep to devDependencies

### PR DESCRIPTION
**Pull request by jrajav**
Fixes #85

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,8 +31,7 @@
     "classnames": "^2.2.5",
     "material-components-web": "^0.27.0",
     "react": "^16.2.0",
```
Comment (jamesmfriedman) [inline package.json]: @jrajav can you do me a favor and also remove react and react-dom from dependencies? They made it in there by accident. These are covered in peer dependencies, so they don't have to be moved, you can just safely delete them.

**Comment by jamesmfriedman** [comment]
#84

File: package.json (hunk 1 above)
Comment (jrajav) [reply package.json]: @jamesmfriedman Done!

**Comment by jamesmfriedman** [comment]
Thanks @jrajav

**Pull request merged**